torchvision/prototype/transforms/_misc.py

```diff
@@ -68,7 +68,7 @@ def forward(self, *inputs: Any) -> Any:
 
         return super().forward(*inputs)
 
-    def _transform(self, inpt: Union[torch.Tensor, features._Feature], params: Dict[str, Any]) -> torch.Tensor:
+    def _transform(self, inpt: TensorImageType, params: Dict[str, Any]) -> torch.Tensor:
```

Comment:
Incorrectly defined `features._Feature` previously. It should have been `features.Image`